transactions/transactions_service.go

```diff
@@ -474,7 +474,7 @@ func (svc *transactionsService) LookupTransaction(ctx context.Context, paymentHa
 	return &transaction, nil
 }
 
-func (svc *transactionsService) ListTransactions(ctx context.Context, from, until, limit, offset uint64, unpaid bool, transactionType *string, lnClient lnclient.LNClient, appId *uint) (transactions []Transaction, err error) {
+func (svc *transactionsService) ListTransactions(ctx context.Context, from, until, limit, offset uint64, pending, unpaid bool, transactionType *string, lnClient lnclient.LNClient, appId *uint) (transactions []Transaction, err error) {
```

Comment:
this introduces another bool and now there is the complexity you see below.

What about we have a `stateType` which and use that. the default would be `TRANSACTION_STATE_SETTLED`

Reply:
But this is problematic because we only want to see pending outgoing payments. This needs more thought. Maybe a function that can alter the query could be passed in :thinking:

Reply:
This is more complicated than I thought, let's move it to the next milestone